rnd/autogpt_builder/src/components/Flow.tsx

```diff
@@ -232,6 +232,20 @@ const FlowEditor: React.FC<{
 
   const onConnect: OnConnect = useCallback(
     (connection: Connection) => {
+      // Check if this exact connection already exists
+      const existingConnection = edges.find(
+        (edge) =>
+          edge.source === connection.source &&
+          edge.target === connection.target &&
+          edge.sourceHandle === connection.sourceHandle &&
+          edge.targetHandle === connection.targetHandle,
+      );
+
+      if (existingConnection) {
+        console.warn("This exact connection already exists.");
```

Comment:
Nice fix! but console.warn on the console might be not very useful for both the developer and the user here, might as well remove this.